datajunction-server/datajunction_server/api/main.py

```diff
@@ -55,6 +55,7 @@ def get_dj_app(
     Get the DJ FastAPI app and optionally inject an OpenTelemetry tracer provider
     """
     settings = get_settings()
+    setup_logging(settings.loglevel)
```

Comment:
Instead of this setup logging function I think it'd be cleaner if we use a [config](https://docs.python.org/3/library/logging.config.html#logging.config.fileConfig) file that the standard logger supports. That will allow for complete configuration of the logger without changing any source code. I can do that in a follow-up PR if you'd like.

Reply:
Good point! I added a `logging.conf` file and then modified this part to configure logging based on that file